module/documents/actor/actor.mjs

```diff
@@ -689,8 +689,8 @@ export default class Actor5e extends Actor {
     // Configure prototype token settings
     const s = CONFIG.DND5E.tokenSizes[this.system.traits.size || "med"];
     const prototypeToken = {width: s, height: s};
-    if ( this.type === "character" ) Object.assign(prototypeToken, {vision: true, actorLink: true, disposition: 1});
-    this.updateSource({prototypeToken});
+    if ( this.type === "character" ) Object.assign(prototypeToken, {sight: { enabled: true }, actorLink: true, disposition: 1});
+    this.updateSource({prototypeToken}); 
```

Comment:
Errant whitespace at the end of this line.

Reply:
Fixed